doc/source/release/1.16.0-notes.rst

```diff
@@ -127,8 +153,18 @@ Array API Standard Support
 Experimental support for array libraries other than NumPy has been added to
 existing sub-packages in recent versions of SciPy. Please consider testing
 these features by setting an environment variable ``SCIPY_ARRAY_API=1`` and
-providing PyTorch, JAX, or CuPy arrays as array arguments.
-Features with support added for SciPy 1.16.0 include:
+providing PyTorch, JAX, or CuPy arrays as array arguments. Many functions
+in `scipy.stats`, `scipy.special`, `scipy.optimize`, and `scipy.constants`
+now provide tables documenting compatible array and device types as well as
+support for lazy arrays and JIT compilation. New features with support and
+old features with support added for SciPy 1.16.0 include:
```

Comment:
```suggestion
old features with support added or enhanced for SciPy 1.16.0 include:
```

Reply:
Feel free to make a separate PR with all enhancements to array API support. This PR only intends to list things that got some sort of support for the first time; otherwise, the list would be very long.